tools/erc-repository-indexer/erc-contract-indexer/src/utils/constants.ts

```diff
@@ -23,4 +23,13 @@ export default {
   GET_CONTRACT_ENDPOINT: '/api/v1/contracts',
   ERC_20_JSON_FILE_NAME: 'erc-20.json',
   ERC_721_JSON_FILE_NAME: 'erc-721.json',
+  NETWORK_REGEX: /^(localnet|previewnet|testnet|mainnet)$/,
+  MIRROR_NODE_URL_REGEX:
+    /^https:\/\/(previewnet|testnet|mainnet)\.mirrornode\.hedera\.com$/,
```

Comment:
Note: mainnet mirror node url is `https://mainnet-public.mirrornode.hedera.com/api/v1/docs/`

Reply:
Okay will update this to mainnet-public in the follow-up PR